project-paths.js

```diff
@@ -0,0 +1,15 @@
+const path = require('path')
```

Comment:
Is this used outside of the scripts dir? if not, can we move it there?

Reply:
im slightly confused by this comment?

Reply:
I was referring to the whole file